Lib/test/test_email/test__encoded_words.py

```diff
@@ -38,6 +38,15 @@ def test_missing_padding(self):
         # 2 missing padding characters
         self._test(b'dg', b'v', [errors.InvalidBase64PaddingDefect])
 
+    def test_urlsafe_alphabet(self):
+        self._test(
+          b'QW5tZWxkdW5nIE5ldHphbnNjaGx1c3MgU_xkcmluZzNwLmpwZw==',
+          b'Anmeldung Netzanschluss S\xfcdring3p.jpg',
+          [errors.InvalidBase64CharactersDefect])
+        # Mix of 2 base64 alphabets
```

Comment:
```suggestion
        # mix of different base64 alphabets
```

I was a bit confused with "2 base64" so let's be a bit more verbose in the comment (in this case the 2 and the 64 from base64 look odd but maybe I'm tired)

Reply:
Agreed!